cabot/metricsapp/tests/test_grafana.py

```diff
@@ -333,12 +333,55 @@ def test_change_queries(self, send_email):
         self.check.queries = self.old_queries
         self.check.save()
 
+        # careful copy/pasting in an IDE - there is whitespace at the end of some lines
```

Comment:
Just remove the whitespace and the comment? would prefer to remove the flake8 issues instead of #noqaing them

Reply:
The whitespace is part of the test string, so if I remove the whitespace the test will fail. Probably should fix the dumper so it doesn't add pointless whitespace...

Reply:
oh nice lol ok this is fine